DailyKitTests/CallParticipantsTests.swift

```diff
@@ -0,0 +1,219 @@
+@testable import DailyKit
+import XCTest
+
+final class CallParticipantsTests: XCTestCase, TestSupport {}
+
+// MARK: - Count
```

Comment:
Suggest rewording these comments slightly for clarity. E.g., removing name, maybe a little more info? "Test participant counts" or "Participant count tests" (or something)

Reply:
`// MARK: - Count` comments a special kind of comments that produces section headers and delimiters in Xcode's symbol dropdown, so keeping them brief is usually preferable:

<img width="302" alt="image" src="https://github.com/daily-demos/daily-ios-starter-kit/assets/138017/c9a9f6f3-e9a9-47ba-92f3-e2c4c59a13ad">

Reply:
Related to the comment above, I moved the `CallParticipants.Builder` to its own file and consolidated the test files to one for each of the two types. I also made the comments a little more descriptive.